lang/pt_BR.json

```diff
@@ -0,0 +1,22 @@
+{
+    "VISION5E.Blindsense": "Sentido Cego",
+    "VISION5E.Burrowing": "Escavação",
+    "VISION5E.DetectEvilAndGood": "Detectar o Bem e o Mal",
+    "VISION5E.DetectMagic": "Detectar Magia",
+    "VISION5E.DetectPoisonAndDisease": "Detectar Veneno e Doença",
+    "VISION5E.DetectThoughts": "Detectar Pensamentos",
+    "VISION5E.DevilsSight": "Visão do Diabo",
+    "VISION5E.DivineSense": "Sentido Divino",
+    "VISION5E.Echolocation": "Ecolocalização",
+    "VISION5E.Ethereal": "Etéreo",
+    "VISION5E.EtherealSight": "Visão Etérea",
+    "VISION5E.GhostlyGaze": "Olhar Fantasmagórico",
+    "VISION5E.Hearing": "Audição",
+    "VISION5E.Inaudible": "Inaudível",
+    "VISION5E.LightPerception": "Percepção da Luz",
+    "VISION5E.SeeInvisibility": "Ver o Invisível",
```

Comment:
I found some weird sources for the pt-BR translation as I didn't have access to the official pt-BR books.
- https://iccarion.fandom.com/pt-br/wiki/Iccarion
- http://www.segredosdealancia.com.br/

I'm not sure what I'm looking at really. The latter uses "Ver Invisibilidade". The former "Ver o Invisível", as well as the translation module (https://gitlab.com/fvtt-brasil/dnd5e/-/blob/master/dnd5e_pt-BR/lang/pt-BR/compendium/dnd5e.spells.json?ref_type=heads#L1022).

Reply:
"Ver o Invisível" is the name of the spell **See Invisibility** in the official books. "Ver Invisibilidade" it's the option **See Invisibility** of the feature **The Third Eye** from the Divination Warlock.

The truth about the translation of D&D5e here in Brazil is: it's shit. So, a lot of guys here make their own translation, because the official is poorly made. That's why you will see the same ability with different names in some sites. The majoraty of brazillians that play 5e, just play it in english.